m2py/utils/post_processing.py

```diff
@@ -361,6 +362,68 @@ def show_classification_correlation(labels, data, data_type, title_flag=True, sa
     pyplot.show()
 
 
+def get_gmm_background_label(gmm_labels, connected_labels):
+    """
+    Gets the GMM backgound comoponent using connected components information. The background component will
+    be the one with the fewest connected components.
+
+    Parameters
+    ---------
+        gmm_labels : NumPy Array
+            GMM labeling
+        connected_labels : NumPy Array
+            connected components labeling
+
+    Returns
+    ----------
+        bg_label : int
+            GMM background label
+
+    """
+
+    return min(list(np.unique(gmm_labels)), key=lambda k: np.size(np.unique(connected_labels[gmm_labels == k])))
+
+
+def applies_morphological_cleanup(labels, bg_label, morph_radius=5, morph_min_size=25):
```

Comment:
This method applies morphological clean-up per band. Use previous method to obtain background label.